client/KeyguardClient.js

```diff
@@ -129,9 +136,11 @@ class KeyguardClient {
      * @param {any[]} [args]
      */
     async _startPopup(requestName, args) {
+        const requestUrl = `${this._keyguardSrc}/request/${requestName}/`;
+
         const $popup = window.open(
-            `${this._keyguardSrc}request/${requestName}/`,
-            'Nimiq Keyguard Popup',
+            requestUrl,
+            this._targetName,
```

Comment:
Mdn says `The name should not contain whitespace.` (https://developer.mozilla.org/en-US/docs/Web/API/Window/open)

Reply:
yep, I remember it breaks in some browsers if you do

Reply:
the `name` is the identifier for this "frame" - the window title will be set by the HTML page loaded inside.